### DISCUSSION TITLE
[Enchancement] Better Landing Page Design

### DISCUSSION THREAD

**Discussion by suryanshsingh2001** [suryanshsingh2001/mockly#30] [Ideas]
### Problem Description
The current landing page could be improved to better engage users and present information more clearly. The existing pricing cards and feature sections do not provide the best user experience or visual appeal.

Example: "The pricing cards are too simplistic, and the feature sections could be better structured to highlight the product’s strengths."

### Desired Enhancement
I would like the landing page to be redesigned with better pricing cards, a more engaging feature section (possibly using a bento grid layout), and an enhanced hero section that grabs user attention right away.

### Alternatives Considered
I explored some alternatives with minimal design changes, but they didn’t significantly improve the page's visual impact or usability. A full redesign focusing on modern UI/UX principles would be more beneficial.

### Possible Implementation
- [ ] Redesign pricing cards with clearer distinctions between tiers, benefits, and CTA buttons
- [ ] Use a bento grid layout for the feature section to create a more modern, visually appealing structure
- [ ] Revamp the hero section with a stronger headline, supporting text, and a CTA
- [ ] Ensure consistent design language and color scheme throughout the page

### Additional Context
Reference websites that inspired the idea for the bento grid feature section:

https://magicui.design/docs/components/bento-grid
https://magicui.design/docs/components/animated-list
---

### Optional Sections:

- **Priority**: High
- **Are you willing to submit a PR for this enhancement?** Yes
- **Does this enhancement require changes in documentation?** No

**Comment by Ehm-Ehs** [comment]
Hi @suryanshsingh2001  i will like to pick up this issue, if you do not mind giving clarity to it

**Comment by suryanshsingh2001** [comment]
> Hi @suryanshsingh2001 i will like to pick up this issue, if you do not mind giving clarity to it

Absolutely! Have a look at the current landing page at [www.mockly.site](http://www.mockly.site/). The feature section definitely has room for improvement. I was thinking of adding a bento grid layout—feel free to refer to the links provided in the issue description for that.

You can also enhance the pricing section's design to make it more visually appealing and informative. You’re welcome to submit separate PRs for the feature section and the pricing section, or combine them into a single PR—it’s up to you!

**Comment by kumariAnjali10** [comment]
Hi suryansh! I also  want to take this issue. Can I ?

**Comment by suryanshsingh2001** [comment]
> Hi suryansh! I also want to take this issue. Can I ?

@kumariAnjali10 These are some issues you can work on. You can add one of these as separate issue. And work on it.
- [ ] Redesign pricing cards with clearer distinctions between tiers, benefits, and CTA buttons
- [ ] Use a bento grid layout for the feature section to create a more modern, visually appealing structure
- [ ] Revamp the hero section with a stronger headline, supporting text, and a CTA
- [ ] Ensure consistent design language and color scheme throughout the page

**Comment by suryanshsingh2001** [comment]
Hey @Ehm-Ehs @kumariAnjali10, this issue has been broken down into two smaller issues #17 and #16. Feel free to comment on those issues, so I can assign you.

**Comment by GitMohit123** [comment]
I'd love to contribute to these enhancements and would appreciate it if you could assign me this task. 

Here's a summary of my observations:
1. Feature Section:
The current layout should be converted into a bento layout grid for a more modern and structured design.
The background contrast could be reduced or removed to maintain consistency.
2. Visibility Issues:
![WhatsApp Image 2024-10-02 at 11 17 51](https://github.com/user-attachments/assets/971bc91d-45b4-4ae1-818e-f96f15a252b5)
Points like "No login" and "No watermarks" are not clearly visible, making them easily ignorable. They could use more prominence for better user engagement.
3. Theme Issues:
![WhatsApp Image 2024-10-02 at 11 16 57](https://github.com/user-attachments/assets/f6b948e6-13e8-4a3a-a740-7cc7a6f6372e)
When switching to the light theme, the image disappears, which should be fixed. The image beside the text could be replaced with an embedded video to make the feature more interactive and easily understandable.
4. Pricing Cards:
The design of the pricing cards could be updated to improve clarity and user experience.
Additionally, there are a few more areas I'd like to review and improve.
5. Revamping the hero section with stronger headline and eye catchy for the users

**Comment by suryanshsingh2001** [comment]
> I'd love to contribute to these enhancements and would appreciate it if you could assign me this task.
> 
> Here's a summary of my observations:
> 
> 1. Feature Section:
>    The current layout should be converted into a bento layout grid for a more modern and structured design.
>    The background contrast could be reduced or removed to maintain consistency.
> 2. Visibility Issues:
>    ![WhatsApp Image 2024-10-02 at 11 17 51](https://private-user-images.githubusercontent.com/154335512/372743692-971bc91d-45b4-4ae1-818e-f96f15a252b5.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MzU2OTgsIm5iZiI6MTcyNzkzNTM5OCwicGF0aCI6Ii8xNTQzMzU1MTIvMzcyNzQzNjkyLTk3MWJjOTFkLTQ1YjQtNGFlMS04MThlLWY5NmYxNWEyNTJiNS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDNUMDYwMzE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzFkOTFkMzUwZWYzMmU0MmUwNTRmOTA4Njc1MzM3OTU0Y2FkNDdmZjVmMTNjYTFjZDZmODRmZGM0NDcwOTkwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Tul_PaEIPS4jE2rtZ9vNOzmaQhQteN2e_4KLZM4X970)
>    Points like "No login" and "No watermarks" are not clearly visible, making them easily ignorable. They could use more prominence for better user engagement.
> 3. Theme Issues:
>    ![WhatsApp Image 2024-10-02 at 11 16 57](https://private-user-images.githubusercontent.com/154335512/372743499-f6b948e6-13e8-4a3a-a740-7cc7a6f6372e.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MzU2OTgsIm5iZiI6MTcyNzkzNTM5OCwicGF0aCI6Ii8xNTQzMzU1MTIvMzcyNzQzNDk5LWY2Yjk0OGU2LTEzZTgtNGEzYS1hNzQwLTdjYzdhNmY2MzcyZS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDNUMDYwMzE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTI4NGYwYTU4Zjk4YzJlZjY3ZjFhNzhiMWM3ZmE0MTMwMDFlZDg0ZjllNWE2YTIxOTMyZWZmNGExZWUyOGIzNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.jx7hFCdVEnMyJopgg1L10WLhXAO9lfW45Aw-Q-zV5vc)
>    When switching to the light theme, the image disappears, which should be fixed. The image beside the text could be replaced with an embedded video to make the feature more interactive and easily understandable.
> 4. Pricing Cards:
>    The design of the pricing cards could be updated to improve clarity and user experience.
>    Additionally, there are a few more areas I'd like to review and improve.
> 5. Revamping the hero section with stronger headline and eye catchy for the users

These are all good insights @GitMohit123 , Feel free to take on any/all of these, create issues for them. And get started with making Mockly a better product.

**Comment by suryanshsingh2001** [comment]
Hey everyone, due to a lot assignees, I am making this issue into discussion thread, feel free to create your own issues with a proper implementation details.

**Comment by suryanshsingh2001** [comment]
Hey everyone!

I’d love to hear your ideas and feedback on improving the landing page. Active participation is key to making Mockly even better, so let’s collaborate and brainstorm together. Every suggestion counts in enhancing the product!

Feel free to share your thoughts!

**Comment by suryanshsingh2001** [comment]
@suryanshsingh2001 @Ehm-Ehs @kumariAnjali10 @GitMohit123 @Laikokwui Hey everyone!
There some new changes made in UI in terms of font style and a simple bento grid. Please feel free to contribute to these issues. Feel free to discuss your ideas here. And try to speed up this process

**Comment by suryanshsingh2001** [comment]
Hey everyone! 👋

We’ve decided to **completely revamp the landing page design** to better reflect Mockly’s vision and improve the overall user experience.  

This is a significant update, and you can track the progress and performance in #49 

Since this is a major change, **I’d really appreciate any feedback or suggestions** you have along the way. Feel free to jump in anytime!

Thanks in advance! 🙌
